opm/output/eclipse/VectorItems/well.hpp

```diff
@@ -0,0 +1,164 @@
+/*
+  Copyright (c) 2018 Equinor ASA
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef OPM_OUTPUT_ECLIPSE_VECTOR_WELL_HPP
+#define OPM_OUTPUT_ECLIPSE_VECTOR_WELL_HPP
+
+#include <vector>
+
+namespace Opm { namespace RestartIO { namespace Helpers { namespace VectorItems {
+
+    namespace IWell {
+        enum index : std::vector<int>::size_type {
+            IHead  =  0, // I-location (one-based) of well head
+            JHead  =  1, // J-location (one-based) of well head
+            FirstK =  2, // Layer ID (one-based) of top/first connection
+            LastK  =  3, // Layer ID (one-based) of bottom/last connection
+            NConn  =  4, // Number of active cells connected to well
+            Group  =  5, // Index (one-based) of well's current group
+            WType  =  6, // Well type
+            WCtrl  =  7, // Well control
```

Comment:
As @atgeirr I found it slightly surprising you were duplicating these constants instead of reusing, but I understand the argument. I guess these will never change - but a comment:
```C++
// NB: These enum values are duplicated here and .... please update both places.
```
would be good.